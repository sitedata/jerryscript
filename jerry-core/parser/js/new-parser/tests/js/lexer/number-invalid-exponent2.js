/* This test contains number literal parser tests. */

.00e000 ; a .e ; 12e-001 ; 7.e5 ; 0.e+09 ;

.633e
