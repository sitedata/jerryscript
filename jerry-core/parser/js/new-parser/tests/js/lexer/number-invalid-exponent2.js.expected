
--- Script parsing start ---

  [  1] CBC_PUSH_LITERAL idx:0->number(.00e000)
  [  0] CBC_POP_BLOCK
  [  1] CBC_PUSH_PROP_LITERAL_LITERAL idx:1->ident(a) idx:2->string(e)
  [  0] CBC_POP_BLOCK
  [  1] CBC_PUSH_LITERAL idx:3->number(12e-001)
  [  0] CBC_POP_BLOCK
  [  1] CBC_PUSH_LITERAL idx:4->number(7.e5)
  [  0] CBC_POP_BLOCK
  [  1] CBC_PUSH_LITERAL idx:5->number(0.e+09)
  [  0] CBC_POP_BLOCK

--- Script parsing end ---

Parse error 'Missing exponent part.' at line: 5 col: 1
